src/components/ArticleCreationForm.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { TextField, Box, Button } from '@material-ui/core'
+import { DropzoneArea } from 'material-ui-dropzone'
+import articleCreationTheme from '../theme/articleCreationTheme'
+
+const ArticleCreationForm = ({ handleSubmit, handleChange }) => {
+  const classes = articleCreationTheme()
+
+  return (
+    <form
+      noValidate
+      autoComplete='off'
+      className={classes.formGroup}
+      data-cy='new-article-modal'
+      onSubmit={handleSubmit}>
+      <TextField
+        className={classes.form}
+        data-cy='article-title'
+        variant='outlined'
+        fullWidth
+        required
+        id='standard-required'
+        label='Title'
+        type='string'
+        name='title'
+        onChange={handleChange}
+      />
+
+      <TextField
+        className={classes.form}
+        data-cy='article-body'
+        variant='outlined'
+        fullWidth
+        required
+        multiline
+        id='standard-required'
+        label='Body'
+        type='text'
+        name='body'
+        onChange={handleChange}
+      />
```

Comment:
One line input for body is not good. Use minRows and maxRows to make TextField multiline, I think it should be really big.

Reply:
It grows the more you write so you can see all the text at once so i believe it's better as it is. Because then you can just scroll upp and down the page to read the article which in my opinion makes it easier to read

Reply:
Ok, lets try how that is working